src/yang/ietf-bgp-policy.yang

```diff
@@ -180,11 +183,12 @@ module ietf-bgp-policy {
           leaf as-path-set-name {
             type string;
             description
-              "name of the AS path set -- this is used to reference the
+              "Name of the AS path set -- this is used to reference the
                set in match conditions";
           }
 
           leaf-list as-path-set-member {
+	    // XXX JMH - This is way too general for more than one reason: is this for a prepend? A specific match?  A regex?  Different containers should be defined for those cases.
```

Comment:
Marked for discussion.

Reply:
Need to have a conversation with Acee about how to tie the logic of what is this module with the routing policy module.

Reply:
Opened an issue and assigned it to Jeff.